### ISSUE TITLE
Font has unwanted ligature. 

### ISSUE THREAD

**Issue by FremyCompany** [jsfiddle/jsfiddle-issues#730]
Initially filed against Firefox; but reproduces on Chrome too.
https://jsfiddle.net/6zqzk9fx/1/

**Comment by zarino** [comment]
I came here to report the same thing. I'm seeing it in Chrome on OS X.

![Three examples of the 'fl' ligature which only occupies a single character width](https://cloud.githubusercontent.com/assets/739624/13704450/f7a5b298-e793-11e5-9d49-0e2760244ea9.png)

It looks like `.CodeMirror` has a font-stack of:

```
font-family: "Fira Mono", Monaco, "Andale Mono", "Lucida Console", "Bitstream Vera Sans Mono", "Courier New", Courier, monospace;
```

The single-character "fl" ligature is coming from Fira Mono. If you disable Fira Mono, you get normal spacing back again. Personally I have no idea why you would want a monospace font with ligatures like this, since the whole point of a monospace font is for each character to align to a single-character grid.

**Comment by oskarkrawczyk** [comment]
Moved to Mozilla's CDN for Fira. Should solve the problem for those who are experiencing it.

**Comment by oskarkrawczyk** [comment]
Also fixes #720
